### ISSUE TITLE
Syntaxnet: Where is the baseline_eval.py

### ISSUE THREAD

**Issue by felixgwu** [tensorflow/models#1211]
I am wondering where the `baseline_eval.py` mentioned in the [CoNLL 2017 document](https://github.com/tensorflow/models/blob/e8464d33d363c6f8e90a29c08e5af11ef01be7a1/syntaxnet/g3doc/conll2017/README.md) is.
I searched through the repo, but couldn't find it.

**Comment by djweiss** [comment]
Looks like that's a dangling reference -- we will fix that and add more up-to-date evaluation instructions. In the meantime, please see evaluator.py, segmenter-evaluator.py, and parser-to-conll.py in the dragnn/tools directory to run evaluations.

**Comment by martinpopel** [comment]
The [official CoNLL2017 evaluation script](https://github.com/ufal/conll2017/blob/master/evaluation_script/conll17_ud_eval.py) can be dowloaded from http://universaldependencies.org/conll17/evaluation.html. Note that unlike the above-mentioned scripts, it handles different tokenization and multi-word tokens.

**Comment by hans** [comment]
Hi @djweiss , I'm not able to use the `evaluator.py` script with the distributed CoNLL 2017 baseline model. (I've tried loading the English and Spanish models; no difference.)

I've reposted the error on a separate GH issue: https://github.com/tensorflow/models/issues/1355#issuecomment-296275929

It seems like the models might be mismatched with the public eval code. Or maybe I'm doing something wrong in the invocation?

I appreciate your help, thanks!
